command/crypto/otp/verify.go

```diff
@@ -67,23 +73,73 @@ as "300ms", "-1.5h" or "2h45m". Valid time units are "ns", "us" (or "µs"), "ms"
 }
 
 func verifyAction(ctx *cli.Context) error {
-	args := ctx.Args()
-	secretFile := args.Get(0)
+	var (
+		secret, algStr string
+		period         uint
+		digits         int
+	)
 
+	secretFile := ctx.String("secret")
 	b, err := ioutil.ReadFile(secretFile)
 	if err != nil {
 		return errs.FileError(err, secretFile)
 	}
-	secret := string(b)
+	secret = string(b)
 	if strings.HasPrefix(secret, "otpauth://") {
-		url, err := otp.NewKeyFromURL(secret)
+		otpKey, err := otp.NewKeyFromURL(secret)
 		if err != nil {
 			return err
```

Comment:
Do we need to wrap here?

Reply:
what? where?

Reply:
errors.Wrap(err, "failed to parse secret")

Reply:
Oh, I see. I thought you were saying that I shouldn't be wrapping. Got confused. Sure, I'll wrap it.